docs/submitting.md

```diff
@@ -78,12 +78,12 @@ Your paper should include:
 
 - A list of the authors of the software and their affiliations, using the correct format (see the example below).
 - A summary describing the high-level functionality and purpose of the software for a diverse, *non-specialist audience*.
-- A *Statement of Need* section that clearly illustrates the research purpose of the software.
+- A *Statement of need* section that clearly illustrates the research purpose of the software and places it in the context of related work.
```

Comment:
@pdebuyl @danielskatz Is there agreement on this wording and it just needs to be massaged into sentence structure elsewhere?

Reply:
It's ok with me

Reply:
yes, that is fine!